evap/contributor/tests/test_views.py

```diff
@@ -1,11 +1,61 @@
 from model_mommy import mommy
 
-from evap.evaluation.models import Course, UserProfile
-from evap.evaluation.tests.tools import ViewTest, create_course_with_responsible_and_editor
+from django.urls import reverse
+from django.core import mail
+
+from evap.evaluation.models import Course, UserProfile, Contribution
+from evap.evaluation.tests.tools import WebTest, ViewTest, create_course_with_responsible_and_editor
 
 TESTING_COURSE_ID = 2
 
 
+class TestContributorDirectDelegationView(WebTest):
+    csrf_checks = False
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.course = mommy.make(Course, state='prepared')
+
+        cls.responsible = mommy.make(UserProfile)
+        cls.non_responsible = mommy.make(UserProfile, email="a@b.c")
+        mommy.make(Contribution, course=cls.course, contributor=cls.responsible, can_edit=True, responsible=True, comment_visibility=Contribution.ALL_COMMENTS)
+
+    def test_direct_delegation_request(self):
+        data = {"delegate_user": self.non_responsible.id}
+        page = self.app.post(reverse('contributor:course_direct_delegate', args=[self.course.id]), params=data, user=self.responsible).follow()
```

Comment:
...maybe we should decide whether we want to use that reverse or not, and then do that everywhere.